frappe/core/page/permission_manager/permission_manager.py

```diff
@@ -77,8 +77,20 @@ def add(parent, role, permlevel):
 
 @frappe.whitelist()
 def update(doctype, role, permlevel, ptype, value=None):
+	"""Update role permission params
+	
+	Args:
+	    doctype (str): Name of the DocType to update params for
+	    role (str): Role to be updated for, eg "Website Manager".
+	    permlevel (int): perm level the provided rule applies to
+	    ptype (str): permission type, example "read", "delete", etc.
+	    value (None, optional): value for ptype, None indicates False
+	
+	Returns:
+	    str: Refresh flag is permission is updated successfully
+	"""
 	frappe.only_for("System Manager")
-	out = update_permission_property(doctype, role, permlevel, ptype, value)
+	out = update_permission_property(doctype, role, permlevel, ptype, value, alert=False)
```

Comment:
Should this alert flag be set through the API as well? I feel we might need this to change in the future. 

Maybe we could default `alert` to False in the API def. How about that?

Reply:
We could, and then set the flag to True in the doctype controllers that utilizes this function.

Reply:
Yeah. `rename_doc` in the module `frappe.model.rename_doc` has a `show_alert` flag that does the same.